OpenSim/Common/STOFileAdapter.h

```diff
@@ -174,6 +174,9 @@ createSTOFileAdapterForReading(const std::string& fileName);
 std::shared_ptr<DataAdapter>
 createSTOFileAdapterForWriting(const AbstractDataTable& table);
 
+typedef STOFileAdapter_<double> STOFileAdapter;
+typedef STOFileAdapter_<SimTK::Vec3> STOFileAdapterVec3;
```

Comment:
I'm fine with this being included in this PR; I'm just curious how it ended up here.